java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -1184,24 +1045,14 @@ public void shouldInitialiseRootPartitionCorrectlyForIntKey() throws StateStoreE
         // Given
         Field field = new Field("key", new IntType());
         Schema schema = Schema.builder().rowKeyFields(field).build();
-        RangeFactory rangeFactory = new RangeFactory(schema);
         StateStore dynamoDBStateStore = getStateStore(schema);
 
         // When
         List<Partition> partitions = dynamoDBStateStore.getAllPartitions();
 
         // Then
         assertThat(partitions).hasSize(1);
```

Comment:
We could remove this assertion, as containsExactly already asserts this with better output.